### PR TITLE
document github labels

### PR DESCRIPTION
**Pull request by jensens**
Take information from 
https://docs.google.com/spreadsheets/d/1IQ73bSQ10b0pwoUFn0u8SE4epDdvwjhTiYTDyPAhx7A/edit#gid=0

Enhanced it and turned into ReST 

@svx @polyester - may you have a look, esp. I'am not sure if the "figure " works in papyrus that way.

### CODE REVIEW

**Comment by svx** [comment]
In index.rst you reference to githublabels, but you added the new file labels.rst, did I oversee something ?

I guess you want to reference in index.rst to labels.rst right ?

**Comment by jensens** [comment]
indeed, fixed

**Comment by svx** [comment]
@jensens you mean the part about 'Main Labels' ?

I just did a test build with mr.docs [don't mind the theme] and in HTML it looks like this:
labels_coredev

**Comment by jensens** [comment]
below the definition list a figure should appear showing the colors of the labels, is it there?

**Comment by polyester** [comment]
Just use the .. img:: directive instead of the figure one

**Comment by jensens** [comment]
ok. I just copied it from somewhere in the docs ... :)

**Comment by svx** [comment]
Oh sorry I did not got that, yes it is working, new test build pic:

label_pic

**Comment by svx** [comment]
@polyester We should update: http://docs.plone.org/about/rst-styleguide.html, currently we not even have a link to the official sphinx-rst docummentation :)

**Comment by svx** [comment]
@jensens @polyester https://github.com/plone/documentation/pull/535

File: docs/labels.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    Normal priority
+14 prio: low
+    Low priority
+
+21 status: confirmed
+    issue was read and got confirmed
+22 status: in-progress
+    somebody works on that issue
+23 status: testing
+    pull request specific, a PR test or similar is running, waiting for results
```
Comment (gforcada) [inline docs/labels.rst]: Do we really need this one? with `in-progress` and `ready` I would say is enough. As long as jenkins is not happy the issue/pull request is in progress.

File: docs/labels.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+24 status: ready
+    pull request specific, tests are passing, merge me asap
+25 status: deferred
+    work on issue or pull request is scheduled for future
+
+31: needs: help
+    anybody lease help with this issue
```
Comment (gforcada) [inline docs/labels.rst]: typo: `please`

File: docs/labels.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+group-prefix
+    It makes things more clear to prefix, similar to a namespace
+
+color
+    while the main types should be easily to differenciate,
+    the other groups are each one color, eventually using a gradient
+
+open/closed
+    This is handled by Github already and does not some extra label!
```
Comment (gforcada) [inline docs/labels.rst]: does not _need_ some extra label ? :-)

**Comment by gforcada** [comment]
@jensens I added some comments if you want to review them

File: docs/labels.rst (hunk 1 above)
Comment (jensens) [inline docs/labels.rst]: indeed, this was introduced before jenkins reported back to github.

**Comment by jensens** [comment]
@gforcada thanks.

**Comment by jensens** [comment]
confused, what happened?

**Comment by gforcada** [comment]
@jensens that a commit in p.a.multilingual with the text `#184` was comitted and the automatic fake commit done by mr.roboto on buildout.coredev made github parse the commit message and found that 184 and thus decided to close the issue :-(

That's why I always put the full URL instead of just the reference :-)

**Pull request merged**